extensions/query-store/src/common/utils.ts

```diff
@@ -75,6 +75,10 @@ export function createSplitView(view: azdata.ModelView, firstComponent: azdata.C
 	return splitview;
 }
 
+/**
+ * These are the available components that can be added to a config dialog.
+ * The sequence of the components here is the order that will be followed for addition to the report.
```

Comment:
oh so this follows the order listed here, even if the array passed into openDialog() is a different order? I would've expected the order to be what's passed in the array. Ex:  passing in [timeIntervalComponent, chartComponent, filterComponent] would display it in that order, rather than [chartComponent, timeIntervalComponent, filterComponent]

Reply:
That makes complete sense. I have updated the logic to follow the order provided in the list.

Reply:
update comment here too with the change :)